cms/templates/settings.html

```diff
@@ -115,14 +115,17 @@ <h3 class="title">${_("Course Summary Page")} <span class="tip">${_("(for studen
           </div>
           % endif
 
-          % if not about_page_editable:
+          % if marketing_enabled:
           <div class="notice notice-incontext notice-workflow">
             <h3 class="title">${_("Promoting Your Course with {platform_name}").format(platform_name=settings.PLATFORM_NAME)}</h3>
             <div class="copy">
               <p>${_(
                 'Your course summary page will not be viewable until your course '
                 'has been announced. To provide content for the page and preview '
                 'it, follow the instructions provided by your Program Manager.')}
+                ${_(
+                'Please note that changes here may take up to a business day to '
+                'appear on your course summary page.')}
```

Comment:
Should we also add in messaging about Publisher if ENABLE_PUBLISHER is true?

Reply:
There is a publisher-side message about the delay in syncing. I figured that might be enough? (And I wasn't sure where to put that message in Studio either...)

Reply:
I wasn't sure exactly where this messaging was, but I kind of thought it would be similar to a tooltip next to the fields that are affected. If that's not easy to do, eh. Just thought it would be good to remind people in both places.

Reply:
The marketing site message I patched is a yellow banner at the top of the page. The individual fields don't have tooltips. And the section headers have only a small amount of space for help text that is already used.

https://studio.edx.org/settings/details/course-v1:UBCx+IndEdu200x+1T2019

I dunno, I need an adult UX person to tell me where.